src/cromshell/abort/command.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+import click
+import requests
+
+from cromshell.utilities import http_utils, io_utils
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command(name="abort")
+@click.argument("workflow_id", required=True, nargs=-1)
+@click.pass_obj
+def main(config, workflow_id):
+    """Abort a running workflow.
+
+    WORKFLOW_ID can be one or more workflow ids belonging to a running workflow
+    separated by a space (e.g. abort [workflow_id1] [[workflow_id2]...]).
+
+    All provided workflow ids must be associated with cromwell url
+    provided in configuration file or in the cli options.
+    """
+
+    LOGGER.info("abort")
+
+    http_utils.assert_can_communicate_with_server(config)
+
+    for wdl_id in workflow_id:
```

Comment:
should this loop be placed in a function?

Reply:
I wouldn't.  You might put the internals of the loop in a function though you wanted to wrap it up somehow.